.travis.yml

```diff
@@ -1,8 +1,6 @@
 language: php
 
 php:
-  - 5.5
-  - 5.6
```

Comment:
@markstory That's just temporary so that I could php 7 jobs running quickly as they are the ones having problem.